prow/plugins/verify-owners/verify-owners_test.go

```diff
@@ -142,7 +142,7 @@ var ownerAliasesFiles = map[string][]byte{
 `),
 }
 
-func IssueLabelsAddedContain(arr []string, str string) bool {
+func IssueLabelsContain(arr []string, str string) bool {
```

Comment:
I don't get why you changed the name of that func?

Reply:
Previously in tests, it worked only with the collection `fghc.IssueLabelsAdded`, now I added test which works with `fghc.IssueLabelsRemoved` therefore I decided maybe more generic name `IssueLabelsContain` would fit better here.